### ISSUE TITLE
use overfitting to our advantage

### ISSUE THREAD

**Issue by Thomas1e** [clemenshelm/chillbill-recognizer#282]
Because there are often similar bills we could use overfitting (specially a high gamma) to our advantage. A high gamma creates some sort of "islands" around the data points. If the data-tuples (of a bill) we want to classify have the same structure as some data-tuples we used  to create the model, the "island" around that specific point will likely contain the new data. This gives us the possibility to make a very likely classification. 

If these process do not return any true combination for a bill, we use another model (with hyperparameters that do not overfitt). 

Overall, we can create several steps (models) from "total overfit" to "none overfitt".
